website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -108,20 +109,55 @@ To delete a published tag to re-tag, follow these steps:
     git push -d origin vx.y.z
 ```
 
-## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules in bamboo
+## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules via bamboo
 
-For testing purposes, you should use your feature branch in cumulus-orca github repo and `prototype-latest` bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `ORCA-test-branch` as the linked repo.
+For testing purposes, you should use your feature branch in cumulus-orca github repo and [`prototype-latest`](https://ci.earthdata.nasa.gov/browse/ORCA-PL) bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `orca test branch` as the linked repo.
 
-Make sure to use the `ORCA Deploy Plan` bamboo plan for deploying the resources. These are the ORCA buckets that are created in disaster recovery AWS account:
+:::warning
+You should reset `feature/ORCA-test-bamboo` before using it.
+1. Rename `feature/ORCA-test-bamboo` to `feature/ORCA-test-bamboo-old`
+1. Create a new branch based off of your branch named `feature/ORCA-test-bamboo`
+1. In the new branch's `bamboo.yaml`:
+   Delete all but one of the plans.
+   Change plan's `name` to `prototype-latest`.
+   In each `repositories` element, change `orca-develop` to `orca test branch`.
+   In each `plan` element, change `OI`/`ODP` in `key` values to `PL`

```

Comment:
This prototype-latest`](https://ci.earthdata.nasa.gov/browse/ORCA-PL) bamboo plan is used for testing purposes.
This is used to test changes in bamboo.yaml before running them in develop branch otherwise you have to merge every change in develop and then run bamboo to test it which is cumbersome

There should not be any rule to delete any plans or change names.
You can say something like "Update spec file as needed for testing"

Reply:
See https://github.com/nasa/cumulus-orca/pull/309/files#r939212998 and https://github.com/nasa/cumulus-orca/pull/309/files#r939215158 for examples of the types of errors this can introduce. `prototype-latest` was effectively a forked branch, out of sync with develop, and not able to accurately represent the state of the bamboo build. Unless it is regularly synced with `develop` (not via manual, prone-to-error updates) it is invalid as a testing environment.

Reply:
in that case, we should probably add a note that 

`once changes are tested and merged into develop, make sure that 'prototype-latest' repo bamboo spec file configuration should be same as the develop branch`

Reply:
This is impossible, as the testing/modification procedure you recommend requires changes to `bamboo.yaml`.